templates/frame.haml

```diff
@@ -91,9 +91,11 @@
       %script{src:"{{ STATIC_URL }}lib/uuid.js"}
       %script{src:"{{ STATIC_URL }}lib/bootstrap-limit.js"}
       %script{src:"{{ STATIC_URL }}lib/jquery-ui-timepicker-addon.js"}
-
-      -if not debug and not testing
-        %script{src:"{{ STATIC_URL }}lib/raven.min.js"}
+        -if not debug and not testing and RAVEN_PUBLIC_DSN
```

Comment:
Don't we need to add RAVEN_PUBLIC_DSN to `temba.orgs.context_processors.settings_includer`? Does this work for you?